fiona/ogrext.pyx

```diff
@@ -1210,6 +1283,85 @@ cdef class WritingSession(Session):
         gdal_flush_cache(cogr_ds)
         log.debug("Flushed data source cache")
 
+    def set_tags(self, tags, ns=None):
+        """Writes a dict containing the dataset or layers's tags.
+        Tags are pairs of key and value strings. Tags belong to
+        namespaces.  The standard namespaces are: default (None) and
+        'IMAGE_STRUCTURE'.  Applications can create their own additional
+        namespaces.
+
+        Parameters
+        ----------
+        tags: dict
+            The dict of metadata items to set.
+        ns: str, optional
+            Used to select a namespace other than the default.
+
+        Returns
+        -------
+        int
+        """
+        cdef GDALMajorObjectH obj = NULL
+        cdef const char *domain = NULL
+
+        if self.cogr_layer != NULL:
+            obj = self.cogr_layer
+        else:
+            obj = self.cogr_ds
+
+        if ns:
+            ns = ns.encode('utf-8')
+            domain = ns
+
+        cdef char **metadata = NULL
+        cdef char *metatata_item_c = NULL
```

Comment:
This appears to be unused.

Reply:
Nice catch! Leftovers from a previous iteration.